test/e2e/logs_test.go

```diff
@@ -63,12 +63,7 @@ func retrieveLog(nodeName, srcPath, destDir string) error {
 	cmd := exec.Command("oc", "adm", "node-logs", "--path="+srcPath, nodeName)
 	out, err := cmd.Output()
 	if err != nil {
-		var exitError *exec.ExitError
-		stderr := ""
-		if errors.As(err, exitError) {
-			stderr = string(exitError.Stderr)
-		}
-		return fmt.Errorf("oc adm node-logs failed with exit code %s and output: %s: %s", err, string(out), stderr)
+		return errors.Wrapf(err, "oc adm node-logs failed with output: %s", string(out))
```

Comment:
Are you planning on opening another PR to use combined output given we still have log collection failures?